### DISCUSSION TITLE
Not building on save

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#12316] [Help]
Hello, I'm relatively new and have been using the Play CDN for a bit, when I decided to use the CLI instead, I ran into some issues.

I followed the steps in https://tailwindcss.com/docs/installation, and set up everything according to the installation guide.

After running 
```npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch```
it *should* build every time I save (cmd+s), instead it runs and works once when running the command it self, and never again.

npm 9.6.7
tailwind 3.3.4
macOS Sonoma

**Comment by wongjn** [comment]
Consider checking the `content` file globs sufficiently cover files that you change and that you'd expect to trigger rebuilds.

**Reply by wongjn** [reply]
> > Consider checking the `content` file globs sufficiently cover files that you change and that you'd expect to trigger rebuilds.
> 
> The content array isn't the issue since, like I said, it does build correctly when I run the command, the problem is that it only builds once when it's supposed to do it every time I save.[^1]

Consider providing a remotely-hosted git repository that reproduces the unexpected behavior, so we can troubleshoot the set up further please.

[^1]: https://github.com/tailwindlabs/tailwindcss/discussions/12316#discussioncomment-7416023

**Reply by ghost** [reply]
I am unable to recreate the behavior outside of my environment

**Comment by ghost** [comment]
> Consider checking the `content` file globs sufficiently cover files that you change and that you'd expect to trigger rebuilds.

The content array isn't the issue since, like I said, it does build correctly when I run the command, the problem is that it only builds once when it's supposed to do it every time I save.

**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/12316#discussioncomment-7416081 to keep the reply chain together.
